### PR TITLE
Document new release schedule

### PR DESCRIPTION
**Pull request by rfratto**
Documents the new release schedule, where we plan to cut a new minor release every 6 weeks. 

I'm not sure if the language here explains the concept well, so please let me know if there's a way to explain it better (particularly around how if one release is moved, releases following it don't move along with it).

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -69,6 +69,18 @@ k3d/Tanka environment. Both examples deploy the Agent, Cortex and Grafana for
 testing the agent. See the [docker-compose README](./example/docker-compose/README.md)
 and the [k3d example README](./example/k3d/README.md) for more information.
 
+## Release schedule
+
+Starting with the v0.28 release on 2022-09-20, a new minor release is planned
+to be created every 6 weeks. This includes a release for Grafana Agent and
+Grafana Agent Operator.
```
Comment (mattdurham) [inline README.md:76]: Mention AgentCtl also?

File: docs/sources/_index.md [hunk 2]
```diff
@@ -21,8 +21,8 @@ There are other ways of sending metrics, logs and traces to the Grafana Stack, G
 
 - Provides a one-stop solution for collecting metrics, logs, and traces.
 - Collects out-of-the-box telemetry from popular projects like MySQL through integrations
-- Works seamlessly with the Grafana Stack. Alternatively, metrics can be sent to any Prometheus-compatible endpoint, and traces can be sent to any OTLP-compatible endpoint.  
-- Offers new solutions to help scale metrics collection like host_filtering and sharding 
+- Works seamlessly with the Grafana Stack. Alternatively, metrics can be sent to any Prometheus-compatible endpoint, and traces can be sent to any OTLP-compatible endpoint.
+- Offers new solutions to help scale metrics collection like host_filtering and sharding
```
Comment (karengermond) [inline docs/sources/_index.md:25]:
```suggestion
- Offers new solutions to help scale metrics collection like host_filtering and sharding.
```

File: README.md [hunk 3]
```diff
@@ -69,6 +69,18 @@ k3d/Tanka environment. Both examples deploy the Agent, Cortex and Grafana for
...
+to be created every 6 weeks. This includes a release for Grafana Agent, Grafana
+Agent Operator, and the `agentctl` binary.
+
+The release schedule is best-effort, and releases may be moved forward or
+backwards if needed. The planned release date for future minor releases are not
+changed if one minor release is moved.
+
+Patch and security releases may be created at any time.
+
 ## Prometheus Vendoring
```
Comment (karengermond) [inline README.md:84]:
```suggestion
## Prometheus vendoring
```

File: docs/sources/_index.md [hunk 4]
```diff
@@ -81,8 +81,8 @@ agents not reporting in.
 For more information on the host filtering mode, refer to the [operation
 guide]({{< relref "./operation-guide#host-filtering" >}}).
 
-### Scraping Service 
-_Scraping Service Mode_ 
+### Scraping Service
```
Comment (karengermond) [inline docs/sources/_index.md:84]:
```suggestion
### Scraping service
```

File: docs/sources/_index.md [hunk 5]
```diff
@@ -105,3 +105,14 @@ Grafana Agent can ingest OpenTelemetry, OpenCensus, Jaeger, Zipkin, or Kafka spa
...
 The Grafana Agent is also capable of exporting to any OpenTelemetry GRPC compatible system.
 
+## Release schedule
+
+Starting with the v0.28 release on 2022-09-20, a new minor release is planned
+to be created every 6 weeks. This includes a release for Grafana Agent, Grafana
+Agent Operator, and the `agentctl` binary.
+
+The release schedule is best-effort, and releases may be moved forward or
+backwards if needed. The planned release date for future minor releases are not
```
Comment (karengermond) [inline docs/sources/_index.md:115]:
```suggestion
backwards if needed. The planned release dates for future minor releases are not
```

**Review by karengermond — APPROVED** [review]
LGMT, minor edits

**Pull request merged**